src/Controller/ModulesController.php

```diff
@@ -65,6 +65,13 @@ public function index() : ?Response
         try {
             $response = $this->apiClient->getObjects($this->objectType, $this->request->getQueryParams());
         } catch (BEditaClientException $e) {
+            // no error flash for autocomplete
+            if (!empty($this->request->getQueryParams()['autocomplete'])) {
```

Comment:
`[minor]` what errors are we catching form `autocomplete` should we hide them?

Reply:
No idea of which errors...possible API php-sdk or server errors. Custom error handling provided on our cakephp web app would render an error on dom element target of ajax call, and we don't want that.

Reply:
ok, we should be able to display errors on ajax call and not hide them => to do in a new issue
I'd probably remove this handling and add a simple comment like: `// ajax calls errors should be displayed....`